edb/lib/_testmode.edgeql

```diff
@@ -97,6 +97,12 @@ ALTER TYPE cfg::AbstractConfig {
         CREATE ANNOTATION cfg::internal := 'true';
         SET default := '';
     };
+
+    CREATE PROPERTY __pg_max_connections -> std::str {
```

Comment:
Hm, is this supposed to be accessed only by tests?

Reply:
Yeah I mean that's my intention - managed Postgres should define/access their own `max_connections` from the Postgres side, or if the Postgres is managed by EdgeDB, we could already configure this value through `max_backend_connections` (for now `pg_settings.max_connections` equals to EdgeDB `max_backend_connections`).